include/common_structs.h

```diff
@@ -1465,8 +1476,8 @@ typedef struct DecorationTable {
     /* 0x8B8 */ s8 decorationType[2];
     /* 0x8BA */ u8 unk_8BA[2];
     /* 0x8BC */ u8 unk_8BC[2];
-    /* 0x8C0 */ s16 unk_8C0[6];
-    /* 0x8BE */ char unk_8BE[30];
+    /* 0x8C0 */ s16 unk_8C0[4];
+    /* 0x8BE */ decorationUnk unk_8C6[2];
 } DecorationTable; // size = 0x8E8
```

Comment:
With the new struct introduced, the size of `DecorationTable` seems to now be `0x8E6`, which might or might not be wrong - I'll leave this to Alex and Ethan to decide

Reply:
Clover has size 8E8, so this is valid

Reply:
I would add padding to the end to make it actually be 8E8

Reply:
Also it looks like the offset comments need to be updated

Reply:
That is a bit outdated from how it looks now
This is how it looks:
    /* 0x8C0 */ s16 unk_8C0[4];
    /* 0x8C6 */ DecorationUnk unk_8C6[2];
} DecorationTable; // size = 0x8E8